lib/iex/test/iex/autocomplete_test.exs

```diff
@@ -86,6 +86,56 @@ defmodule IEx.AutocompleteTest do
     :code.delete(Sample)
   end
 
+  describe "Elixir no completion for default argument functions with doc set to false" do
+    test "real Elixir case" do
```

Comment:
well. those functions have only one default argument, and I want to be sure we are building the right list. So I will look for a function that has two default arguments, or create that only one.